src/UltraStar-Creator.pro

```diff
@@ -77,10 +77,11 @@ INCLUDEPATH += ../include/bass \
 	../include/srtparser
 
 win32 {
-	INCLUDEPATH += ../include/taglib
+	INCLUDEPATH += "C:/Program Files/taglib/include/taglib"

```

Comment:
Since the headers and lib aren't part of the repo anymore, I changed the paths to the default install location for taglib.
Alternative would be to expect them in some folder within the repo, and have the CI copy the files from the freshly built taglib install there. This seems a bit arbitrary though, so I think I'd prefer it this way.

Reply:
To be honest, I don't know which of the two would be best practice, so I am fine with your suggestion. Thanks a lot!